### PR TITLE
correct copyright

### PR DESCRIPTION
**Pull request by paolo-caroni**

### CODE REVIEW

**Comment by rzr** [comment]
Thanks,

If you click on the  tick icon it will lead you to:

https://travis-ci.com/rzr/pinball-table-hurd/builds/143807610

there you'll see some warnings:

``
Now running lintian pinball-table-hurd_0.0.0-1_amd64.changes ...

warning: the authors of lintian do not recommend running it with root privileges!

W: pinball-table-hurd source: missing-license-text-in-dep5-copyright CC-BY-SA-2.0 (paragraph at line 61)

W: pinball-table-hurd source: missing-license-text-in-dep5-copyright CC-BY-SA-3.0 (paragraph at line 64)

W: pinball-table-hurd source: file-without-copyright-information .dockerignore

W: pinball-table-hurd source: file-without-copyright-information .travis.yml

W: pinball-table-hurd source: file-without-copyright-information .whitesource

W: pinball-table-hurd source: file-without-copyright-information Dockerfile

W: pinball-table-hurd source: file-without-copyright-information Makefile.am

W: pinball-table-hurd source: file-without-copyright-information README.md

W: pinball-table-hurd source: file-without-copyright-information acinclude.m4

W: pinball-table-hurd source: file-without-copyright-information bootstrap

W: pinball-table-hurd source: file-without-copyright-information configure.ac

W: pinball-table-hurd source: file-without-copyright-information data/hurd/Makefile.am

W: pinball-table-hurd source: file-without-copyright-information data/hurd/README.txt

W: pinball-table-hurd source: file-without-copyright-information data/hurd/floor-room.jpg

W: pinball-table-hurd source: file-without-copyright-information data/hurd/local.mk

W: pinball-table-hurd source: file-without-copyright-information data/hurd/up.wav

W: pinball-table-hurd source: file-without-copyright-information data/hurd/wood.jpg

W: pinball-table-hurd source: file-without-copyright-information docker-compose.yml

W: pinball-table-hurd source: missing-license-paragraph-in-dep5-copyright cc-by-sa-2.0 (paragraph at line 24)

W: pinball-table-hurd source: missing-license-paragraph-in-dep5-copyright cc-by-sa-3.0 (paragraph at line 32)



```


No need to list all files, 
we can assume that default (*) files are GPL-2.0+

File: debian/copyright [hunk 1]
```diff
@@ -35,7 +35,7 @@ Copyright: 2010 Paolo Caroni <kenren89@gmail.com>
            1997 Stephen McCamant
 License: CC-BY-SA-3.0
 
-       data/hurd/freesoftwaresong.xml
+Files: data/hurd/freesoftwaresong.xml
```
Comment (rzr) [inline debian/copyright]: Files: data/hurd/freesoftwaresong.*

should work too

**Review by rzr — COMMENTED** [review]
Feel free to push over this PR, and check travis logs

**Comment by rzr** [comment]
rebase will be needed, 
you can "skip" commits which are not yours, 
then I will merge and update deb file

Thanks again

**Comment by paolo-caroni** [comment]
I have rebased, all checks have passed.
However there are some conflicts, I have manually correct it, I hope is all right

**Comment by rzr** [comment]
There are still conflicts to solve.

what you can try to do:

```sh
git branch sandbox/$USER/bad/master # backup in case of
git fetch --all 
git reset remotes/upstream/master # catching up
git diff # see lot of changes that are not needed
git difftool # only keep relevant changes
git diff # see only changes in debian/copyright file
git push -f 
```

**Pull request closed without merging**